sdk/cosmos/tests/cosmos_database.rs

```diff
@@ -12,9 +13,10 @@ async fn create_and_delete_database() {
     let client = setup::initialize().unwrap();
 
     // list existing databases and remember their number
-    let databases = client
-        .list_databases(Context::new(), ListDatabasesOptions::new())
+    let databases = Box::pin(client.list_databases(Context::new(), ListDatabasesOptions::new()))
+        .next()
         .await
+        .unwrap()
```

Comment:
This one fails with:

```
---- create_and_delete_database stdout ----
thread 'create_and_delete_database' panicked at 'called `Option::unwrap()` on a `None` value', sdk/cosmos/tests/cosmos_database.rs:19:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I'll investigate further and get back to you, for the moment I'll revert the review.